src/commands/site-list.php

```diff
@@ -167,32 +212,158 @@ function( $site ) {
 		}
 	}
 
-	protected function count_sites( $site_list, $term ) {
+	protected function eval_which_host( $site, $pressable_sites ) {
+		if ( true === $site->is_wpcom_atomic ) {
+			$server = 'Atomic';
+		} elseif ( true === $site->jetpack ) {
+			if ( in_array( parse_url( $site->URL, PHP_URL_HOST ), $pressable_sites ) ) {
```

Comment:
I'd add `true` as a third argument here to force strict comparison. Just to make sure there are no undesirable results for older PHP versions.

Reply:
Added in https://github.com/Automattic/team51-cli/pull/159/commits/1485ef2f75063d0791fd95d6c45ab34a20937084